workspaces/cli-client/src/spec-service-client.ts

```diff
@@ -51,12 +52,19 @@ export class Client implements ISpecService {
     );
   }
 
-  loadConfig(): Promise<{ config: IApiCliConfig }> {
-    return JsonHttpClient.getJson(
+  async loadConfig(): Promise<{ config: IApiCliConfig; configRaw: string }> {
+    return await JsonHttpClient.getJson(
```

Comment:
awaiting should be up to the caller, this should just pass the promise through. Same for all of these, to be consistent.